guidelines.md

```diff
@@ -101,3 +101,61 @@ As an example, consider minimum and maximum length of the reservation:
+  - Select **see more** > **ok**.
+    - If you're having trouble, check the [Mews guide](https://help.mewssystems.com/reference#about-marketplace) for more information on setting up the connection.
+    - At this point, you'll now be viewing the active connections for the demo environment. Select Settings for the connection you've just created. 
+    - **Name:** name of your application/company.
+    - Select the **Enabled** checkbox, and click **Save**.
+      - If the connection is not enabled, you will not receive any data from the property.
+  - Add the unique access token (from the first step) to your code.
+
+### Certification Procedure
+
+All testing must be conducted in the [demo environment](https://mews-systems.gitbook.io/connector-api/guidelines#authentication) before going live.
+
+Once you have completed building and testing the connection, send an email to marketplace@mewssystems.com with the following information:
+
+- Name of application.
+- Type of application (e.g. Revenue management, Point of sales etc.)
+- A list of all the intended API endpoints used, as well as the logic (use case) for each endpoint.
+- Complete this [commercial form](https://goo.gl/forms/lCs75PuCuszhgp0L2) (sales & technical contacts, description of application/company, etc.) 
+  - This form is mandatory for certification.
+- Once the Marketplace team has this information, you will need to run your connection as you would in a live context for 24 hours.
+
+### Evaluation
+
+- If we notice any issues with your connection, you'll be contacted with questions &/or requests for improvements.
+- If the required tests can't be completed successfully, Mews will not allow the connection to advance to the production environment.
+  - If no issues are found, they must be resolve them before the certification process can be completed.
+- If there are no critical issues discovered during certification, we'll advance your application to the production environment.
+- Before you go live, Mews will issue a new [Client token](https://mews-systems.gitbook.io/connector-api/guidelines#requests) for use in the production environment. The token will be sent to the technical contact email that you provide your commercial form.
+- Once that is complete, you will become Mews certified
+
+### Production Authentication
+
+Once you're Mews certified, you'll be added to the Mews Marketplace. You'll then be able to operate your application in the [Mews production environment](https://mews-systems.gitbook.io/connector-api/guidelines#production-environment), which gives you the ability to work wit any Mews property that would like a connection with your system.
+
+**Client Token**
+
+- In order to connect with Mews, you (the partner) will be assigned a unique client token. Please hardcode this token within your own system. 
+- A single email address will be added as a Client token contact. If you'd like to change this address, please contact marketplace@mewssystems.com
```

Comment:
Is it necessary to have this information here? How is it important during a certification that you can have only one email?

Reply:
It is important to have this information as a large amount of clients often get confused between an access and client token when going live. It is common for them to swap the access and client token for live implementations.